lib/MediaConnection.js

```diff
@@ -0,0 +1,57 @@
+const Utils = require('./Utils');
+const Grammar = require('./Grammar');
+const debugerror = require('debug')('JsSIP:ERROR:Socket');
+
+debugerror.log = console.warn.bind(console);
+
+/**
```

Comment:
I would suggest just removing this comments and pointing to the interface definition.